jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,22 @@
+import socket
+import sys
+
+port = 5001 
+
+def main():
+	try:
+		client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+		client_socket.connect(("localhost", port))
```

Comment:
"localhost" should be also stored in a constant. For example: HOST = "localhost"